src/test/com/fulcrologic/rad/database_adapters/sql_test.clj

```diff
@@ -0,0 +1,70 @@
+(ns com.fulcrologic.rad.database-adapters.sql-test
+  (:require
+    [com.fulcrologic.rad.attributes            :as rad.attr :refer [defattr]]
+    [com.fulcrologic.rad.database-adapters.sql :as rad.sql]
+    [clojure.test :refer [deftest testing is]]))
+
+(defattr account-id :account/id :uuid
+  {::rad.attr/identity? true
+   ::rad.sql/schema :production
+   ::rad.sql/tables #{"accounts"}})
+
+(defattr account-name :account/name :string
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"accounts"}})
+
+;; Derived data
+(defattr account-locked? :account/locked? :boolean
+  {})
+
+(defattr user-id :user/id :uuid
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"users"}})
+
+(defattr user-name :user/name :string
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"users"}})
+
+(deftest delta->txs-test
```

Comment:
I strongly support tests, but I'm likely to be a little particular/critical :)

One comment is just that it might not be best to write too many tests until the API shape stabilizes a bit more...but I will give some other feedback as things stand out.